api/MediaQueryList.json

```
@@ -29,10 +29,12 @@
             "version_added": true
           },
           "safari": {
-            "version_added": "5.1"
+            "version_added": "5.1",
+            "notes": "In Safari, <code>MediaQueryList</code> is not based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>remoteListener()</code> to observe media query lists. This is expected to change in macOS 11 Big Sur."
           },
           "safari_ios": {
-            "version_added": "5"
+            "version_added": "5",
+            "notes": "In Safari, <code>MediaQueryList</code> is not based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>remoteListener()</code> to observe media query lists. This is expected to change in iOS 14."
```

Similar issues here. Probably one of these is more accurate:

More certain:

```suggestion
            "notes": "Before iOS 14, <code>MediaQueryList</code> is not based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>remoteListener()</code> to observe media query lists."
```

More speculative:

```suggestion
            "notes": "<code>MediaQueryList</code> is not based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>remoteListener()</code> to observe media query lists."
```